.github/workflows/pull-requests.yaml

```
@@ -55,35 +56,149 @@ jobs:
-    runs-on: [self-hosted]
-    needs: build
 
-    # Never run when the PR carries the "release" label.
-    if: |
-      !contains(github.event.pull_request.labels.*.name, 'release')
+  resolve_assets:
+    name: "Resolve assets"
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'release')
+    outputs:
+     installer_id: ${{ steps.fetch_assets.outputs.installer_id }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 5

(indentation)